svc/shows/transport.go

```diff
@@ -202,6 +202,7 @@ func decodeAddShowRequest(_ context.Context, r *http.Request) (interface{}, erro
 		Cover:         r.URL.Query().Get(cover),
```

Comment:
Use HTTP methods according to REST. 
- add = POST
- update whole entity = PUT
- update partially = PATCH
All these methods have a body, and payload must be taken from it